database/migrations/2024_04_09_115053_create_appointments_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('appointments', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained('users')->cascadeOnDelete();
+            $table->foreignId('patient_id')->constrained('patients')->cascadeOnDelete();
+            $table->string('service_type');
+            $table->dateTime('appointment_date_begin');
+            $table->dateTime('appointment_date_end');
+            $table->string('appointment_type');
```

Comment:
Same here. You are in the appointments table, so you don't need the appointment prefix in the columns.